release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -0,0 +1,861 @@
+[[release-notes-ocp-3-11-release-notes]]
+= {product-title} 3.11 Release Notes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Red Hat {product-title} provides developers and IT organizations with a hybrid
+cloud application platform for deploying both new and existing applications on
+secure, scalable resources with minimal configuration and management overhead.
+{product-title} supports a wide selection of programming languages and
+frameworks, such as Java, Javascript, Python, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Kubernetes, {product-title} provides a
+secure and scalable multi-tenant operating system for today’s enterprise-class
+applications, while providing integrated application runtimes and libraries.
+{product-title} enables organizations to meet security, privacy, compliance, and
+governance requirements.
+
+[[ocp-311-about-this-release]]
+== About This Release
+
+Red Hat {product-title} version 3.11
+(link:https://access.redhat.com/errata/RHBA-2018:2652[RHBA-2018:2652]) is now
+available. This release is based on
+link:https://github.com/openshift/origin/releases/tag/v3.11.0-alpha.0[OKD 3.11] and
+it uses Kubernetes 1.11. New features, changes, bug fixes, and known issues that
+pertain to {product-title} 3.11 are included in this topic.
+
+{product-title} 3.11 is supported on RHEL 7.4 and 7.5 with the latest packages
+from Extras, including Docker 1.13. It is also supported on Atomic Host 7.4.5
+and newer.
+
+For initial installations, see the
+xref:../install/index.adoc#install-planning[Installing Clusters] documentation.
+
+To upgrade to this release from a previous version, see the
+xref:../upgrading/index.adoc#install-config-upgrading-index[Upgrading Clusters]
+documentation.
+
```

Comment:
Are we adding a note that 3.11 will be the last release in the 3.x stream and that large changes to the underlying architecture and installation process are coming in 4.0?

We're deprecating the blue/green install method in 4.0, and I'd like to announce that along with whatever we're saying for 4.0.

Reply:
@kalexand-rh Noted it in there now. Thanks!